com/runcmd.exp

```diff
@@ -10,6 +10,12 @@
 #								#
 #################################################################
 #
+#
+#	USAGE:
+#		runcmd.exp <cmd>
+#			where cmd is a string of ; delimited commands
+#
+
 set timeout 60
```

Comment:
The framework level timeout of 60 seconds might not be enough depending on the input script to run, particularly on slower systems. Make it 3600 seconds (1 hour) with a comment as to why it is huge.